README.md

```diff
@@ -236,98 +241,75 @@ brew update; brew install kubernetes-helm
 
 ```bash
 helm init
+kubectl create clusterrolebinding tiller-clusterrole-binding --clusterrole=cluster-admin --serviceaccount=kube-system:tiller
 ```
 
+*Note:* This setup will deploy the helm server component and will give helm access to the whole cluster. For a more secure, customized setup, please read -> https://helm.sh/docs/using_helm/#role-based-access-control
+
 ### 7. Add the Alfresco Incubator Helm Repository
 
 ```bash
 helm repo add alfresco-incubator https://kubernetes-charts.alfresco.com/incubator
+helm repo add alfresco-stable https://kubernetes-charts.alfresco.com/stable
 ```
 
 ### 8. Add Local DNS
 
-Add Local DNS Entry for Host Machine (needed for JWT issuer matching). Be sure to specify an active network interface.  It is not always `en0` as illustrated.  You can use the command `ifconfig -a` to find an active interface.
+We will be forming a local dns with the use of nip.io. All you have to do it get your ip using the following command.
 
 ```bash
-sudo sh -c 'echo "`ipconfig getifaddr en0`       localhost-k8s" >> /etc/hosts'; cat /etc/hosts
+ipconfig getifaddr en0
 ```
 
-*Note:* If your IP address changes you will need to update the `/etc/hosts` entry for localhost-k8s.
+*Note:* Save this ip for later use.
 
 ### 9. Docker Registry Pull Secrets
 
 See the Anaxes Shipyard documentation on [secrets](https://github.com/Alfresco/alfresco-anaxes-shipyard/blob/master/SECRETS.md).
 
 *Note*: You can reuse the secrets.yaml file from charts/incubator directory.  
 
-### 10. Apply Alfresco Process Services license
+### 10. Download and modify the minimal-values.yaml file
+
+The minimal-values.yaml file contains values for local only development and multiple components are disabled with the purpose of reducing the memory footprint of the Digital Business Platform. This should not be used as a starting point for production use.
+
+Pull the minimal values file from the current repo:
 
-If you have a valid Alfresco Process Services license, you can apply it at deployment time 
-by creating a secret called `licenseaps` from your license file:
 ```bash
-kubectl create secret generic licenseaps --from-file=./activiti.lic --namespace=$DESIREDNAMESPACE
+curl -O https://raw.githubusercontent.com/Alfresco/alfresco-dbp-deployment/master/charts/incubator/alfresco-dbp/minimal-values.yaml
 ```
 
-This step is optional. If you choose not to deploy the license this way,
-Alfresco Process Services will start up in read-only mode and you will need to apply it manually (see [Notes](#12-check-dbp-components) below).
+Open it in your favorite text editor and replace all occurences of REPLACEME with the IP you previously got from step 8.
```

Comment:
Here as well, can we have a `sed` command?

Reply:
Someone will have so much fun with the windows script :)